src/state/slices/opportunitiesSlice/selectors/combined.ts

```diff
@@ -196,20 +196,21 @@ export const selectAggregatedEarnOpportunitiesByProvider = createDeepEqualOutput
       if (cur.type === DefiType.Staking) {
         acc[provider].opportunities.staking.push(cur.id)
         const stakingOpportunity = cur as StakingEarnOpportunityType
-        const rewardsAmountFiat = stakingOpportunity.isClaimableRewards
-          ? Array.from(stakingOpportunity.rewardAssetIds ?? []).reduce((sum, assetId, index) => {
-              const asset = assets[assetId]
-              if (!asset) return sum
-              const marketDataPrice = marketData[assetId]?.price
-              const cryptoAmountPrecision = bnOrZero(
-                stakingOpportunity?.rewardsAmountsCryptoBaseUnit?.[index],
-              ).div(bnOrZero(10).pow(asset?.precision))
-              return bnOrZero(cryptoAmountPrecision)
-                .times(marketDataPrice ?? 0)
-                .plus(bnOrZero(sum))
-                .toNumber()
-            }, 0)
-          : '0'
+        const rewardsAmountFiat = Array.from(stakingOpportunity.rewardAssetIds ?? []).reduce(
```

Comment:
Correct, the previous bit has been reverted as part of this PR so it can be tackled holistically instead! https://github.com/shapeshift/web/pull/4098/commits/79faf8e484b407a8ea710ba95ff792cd2e239774